.soliumrc.json

```diff
@@ -6,6 +6,7 @@
   "rules": {
     "security/no-block-members": "off",
     "security/no-inline-assembly": "off",
+    "no-empty-blocks": "off",
```

Comment:
Same here, does it make more sense to disable the lint locally where this happens?

Reply:
Well ya. That is what I had before and I can easily make this the case again. Good catch.

Reply:
In fact, I don't even think we need to worry about empty blocks anyway so I would argue that it be removed from solhint completely... (since off is the same as not mentioned)